integration-tests/src/test/java/com/datastax/oss/driver/api/core/config/DriverExecutionProfileIT.java

```diff
@@ -107,11 +103,8 @@ public void should_use_profile_request_timeout() {
   public void should_use_profile_default_idempotence() {
     DriverConfigLoader loader =
         SessionUtils.configLoaderBuilder()
-            .withProfile(
-                "idem",
-                DefaultDriverConfigLoaderBuilder.profileBuilder()
-                    .withBoolean(DefaultDriverOption.REQUEST_DEFAULT_IDEMPOTENCE, true)
-                    .build())
+            .startProfile("idem")
```

Comment:
We don't have unit test for two profiles set programmatically

Reply:
It's indirectly covered in integration tests. But yes, I can add unit tests for that too.